Added CommunityToolKit AvatarView documentation. (#117)
MicrosoftDocs/CommunityToolkit: docs/maui/views/index.md

| View | Description |
| --------- | ----------- |

> | [`AvatarView`](AvatarView.md) | The `AvatarView` is a control for displaying an image or text initials avatar. |

Reviewer: As per the sugestion for description of `AvatarView.md`

```suggestion
| [`AvatarView`](AvatarView.md) | The `AvatarView` is a control for displaying a user's avatar image or their initials. |
```
